pkgs/misc/lilypond/with-fonts.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lilypond, lndir }: fonts: stdenv.mkDerivation {
```

Comment:
Could be default value to just use all fonts available in nixpkgs? Some people just want to get something working without having to override a package.